.azure/pipelines/ci.yml

```diff
@@ -38,7 +38,12 @@ variables:
     value: ''
   - name: _SignType
     value: ''
+  - name: _UseHelixOpenQueues
```

Comment:
I'm certain this approach works, but just wanted to suggest that a simpler one might be to conditionally declare lists the way it's done in the runtime repos instead of a special property.

Reply:
@MattGal do you mean the PR should instead detect that `$(HelixApiAccessToken)` is non-empty?

Reply:
I'll file a tracking issue to consider revamping the queue definitions into lists instead of representing each queue as an object which I think was the intent originally

Reply:
Still wonder why we need the `_UseHelixOpenQueues` variable and `$(UseHelixOpenQueues)` property when `$(HelixApiAccessToken)` is empty in public builds.

Reply:
Because a variable named to logically signal intent is much easier to understand than trying to inference what HelixApiAccessToken means, this build stuff is impossible to read already, I think its worth an extra variable to make it clear what logic is specifically dealing with open/internal helix queues.  The tracking issue isn't about this, so much as refactoring how we define the set of helix queues to use and how we logically represent the notion of a helix queue

Reply:
Filed https://github.com/dotnet/aspnetcore/issues/18991 to track revisiting how we deal with queue definitions